vespalib/src/vespa/vespalib/util/threadstackexecutorbase.h

```diff
@@ -78,16 +80,20 @@ class ThreadStackExecutorBase : public SyncableThreadExecutor,
 
     struct BlockedThread {
         const uint32_t wait_task_count;
-        Monitor monitor;
+        mutable std::mutex monitor;
+        mutable std::condition_variable cond;
         bool blocked;
         BlockedThread(uint32_t wait_task_count_in)
-            : wait_task_count(wait_task_count_in), monitor(), blocked(true) {}
+            : wait_task_count(wait_task_count_in), monitor(), cond(), blocked(true) {}
         void wait() const;
         void unblock();
     };
 
     std::unique_ptr<FastOS_ThreadPool>   _pool;
-    Monitor                              _monitor;
+    mutable std::mutex                   _monitor;
```

Comment:
Prefer calling this one '_lock' as the other places, since a monitor is both a lock and a condition_variable

Reply:
Done, I did no renaming to make it easier to see the changes.